cdc/sink/mysql.go

```diff
@@ -1064,7 +1068,12 @@ func (s *mysqlSink) prepareDMLs(rows []*model.RowChangedEvent, replicaID uint64,
 			}
 		}
 
-		// Case for insert event or update event
+		// Case for update event or insert event.
+		// For update event:
+		// If old value is disabled or in safe mode, update will be translated to DELETE + REPLACE SQL.
+		// So we will prepare a REPLACE SQL here.
+		// For insert event:
+		// It will be translated directly into a INSERT(old value is enabled and not in safe mode)/REPLACE(old value is disabled or in safe mod) SQL.
```

Comment:
```suggestion
		// It will be translated directly into a INSERT(old value is enabled and not in safe mode)/REPLACE(old value is disabled or in safe mode) SQL.
```

Reply:
Better to wrap in another line.

Reply:
Fixed.